packages/recommended/src/index.ts

```diff
@@ -26,6 +26,10 @@ export const bootstrapRecommendedConfiguration = () => {
   RuleRegistryInstance.register("RedundantImportsDetector", () => new RedundantImportsDetector());
   RuleRegistryInstance.register("ESLintPreferConstRule", (l: Logger) => new ESLintPreferConstRule(l));
   RuleRegistryInstance.register("NoRedundantDepsRule", (l: Logger) => new NoRedundantDepsRule(l));
+  RuleRegistryInstance.register(
+    "EnforceRationale",
+    (l: Logger, options?: EnforceRationaleOptions) => new EnforceRationale(l, options)
```

Comment:
I don't think `options` will ever be populated. Probably shouldn't be in this callback signature.

Reply:
Is there currently a way to plumb through options specified in the boll config for a particular rule to the instantiation of the rule itself? That's what I'd like to be doing here but I wasn't certain if there was a method for doing that just yet.